spec/functional/run_lock_spec.rb

```diff
@@ -427,6 +433,8 @@ def create_lock
       end
     end
 
+    attr_writer :run_lock
```

Comment:
What's this for? I don't see `self.run_lock = thing` elsewhere in the diff, or see any `@run_lock` references.